docs/docs/configuration.md

```
@@ -72,6 +72,15 @@ Configuring the Tuva EMPI backend is primarily done with a JSON file:
 }
 ```
 
+    // Matching service settings (required)
```

markdownlint-cli2 (0.18.1)

Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)